src/datascience-ui/data-explorer/reactSlickGridFilterBox.tsx

```diff
@@ -3,11 +3,26 @@
 'use strict';
 
 import * as React from 'react';
+import { IIconProps, initializeIcons, SearchBox } from '@fluentui/react';
+
+initializeIcons();
```

Comment:
I thought this needs to be done one place, if thats the case then I'd move it to the entry point of our UI (so that we dont' have multiples of these)

Reply:
Moved to index.tsx, thanks!